internal/core/entity/entity.go

```diff
@@ -0,0 +1,200 @@
+package entity
+
+import (
+	"time"
+
+	"github.com/api7/apisix-seed/internal/utils"
+)
+
+type BaseInfo struct {
+	ID         string `json:"id"`
+	CreateTime int64  `json:"create_time,omitempty"`
+	UpdateTime int64  `json:"update_time,omitempty"`
+}
+
+func (info *BaseInfo) GetBaseInfo() *BaseInfo {
+	return info
+}
+
+func (info *BaseInfo) Creating() {
```

Comment:
As this project only rewrites the etcd data, there is no need to provide a Create method. The snowflake dependency can be removed too.

Reply:
I will remove some unnecessary code.